src/screens/VaultScreen.js

```diff
@@ -37,22 +40,53 @@ export default class VaultScreen extends React.Component {
           upvotes: 0,
           category: 2,
           id: 1,
+          veteran_has_upvoted: boolean
         },
         */
-      ]
+      ],
+      stillLoading: true,
     };
     this.falseState = this.falseState.bind(this);
     this.setOpposite = this.setOpposite.bind(this);
     this.renderResourceContent = this.renderResourceContent.bind(this);
   }
 
   componentDidMount() {
+    this.retrieveResources();
+  }
+
+  async retrieveResources() {
+    try {
+      const endpoint = APIRoutes.resourcePath();
+      let response_json = await BaseRequester.get(endpoint);
+      data = response_json.map(function(item) {
+        return {
+          id: item.id,
+          title: item.file_name,
+          date: this.formatDate(item.updated_at),
+          link: item.url,
+          partner_org: item.owner_id,
+          description: item.description,
+          category: this.getCategory(parseInt(item.category)),
+          upvotes: item.num_upvotes,
+          veteran_has_upvoted: item.veteran_has_upvoted,
+        };
+      }, this);
+      this.setState({ resources: data, stillLoading: false });
+      this.setState({ stillLoading: false });
+      return Promise.resolve(response_json);
+    } catch (error) {
+      return Promise.reject(error);
+    }
+  }
 
+  formatDate(date) {
+    return date.substring(5, 7) + "/" + date.substring(8, 10) + "/" + date.substring(0, 4)
```

Comment:
is it possible to keep this as a JS Date object? https://docs.microsoft.com/en-us/scripting/javascript/date-and-time-strings-javascript